rules/promotions/elastic_endpoint.toml

```diff
@@ -32,6 +32,7 @@ event.kind:alert and event.module:(endpoint and not endgame)
 
 [[rule.exceptions_list]]
 id = "endpoint_list"
+list_id = "endpoint_list"
```

Comment:
Note: The schema for `exception_list` is currently just an array, but if there is some structure behind it, we should update it to reflect what is expected in the API. We can handle this in a future PR though.